lib/prism/translation/parser/lexer.rb

```diff
@@ -413,7 +413,7 @@ def to_a
                 while token.type == :STRING_CONTENT
                   current_length += token.value.bytesize
                   # Heredoc interpolation can have multiple STRING_CONTENT nodes on the same line.
-                  is_first_token_on_line = lexed[index - 1] && token.location.start_line != lexed[index - 2][0].location&.start_line
+                  is_first_token_on_line = lexed[index - 1] && token.location.start_line != lexed[index - 2]&.first&.location&.start_line
```

Comment:
Can we avoid the mix of index accesses and `first` with something like this?

```suggestion
                  prev_prev_token, _ = lexed[index - 2] if index - 2 >= 0
                  is_first_token_on_line = lexed[index - 1] && token.location.start_line != prev_prev_token&.location&.start_line
```